www/js/survey/multilabel/MultiLabelButtonGroup.tsx

```diff
@@ -151,16 +191,47 @@ const MultilabelButtonGroup = ({ trip, buttonsInline = false }) => {
             </Dialog.Title>
             <Dialog.Content style={{ maxHeight: windowHeight / 2, paddingBottom: 0 }}>
               <ScrollView style={{ paddingBottom: 24 }}>
+                <Text style={{ fontSize: 12, color: '#333333', paddingVertical: 4 }}>
```

Comment:
We have a dynamic theme system where all the colors are derived from the theme defined in `/js/appTheme.ts`.
Instead of inline hex codes please use the various properties of `colors`, like `colors.onBackground` or `colors.onSurface`
The reason for doing this is 1) it's tidy and 2) if we want to change something it's easier to modify or swap themes from one location rather than scattered across the codebase

More info on this: https://callstack.github.io/react-native-paper/docs/guides/theming/#theme-properties